src/njoy21/input/ACER/Card7/Thin1.hpp

```diff
@@ -0,0 +1,20 @@
+struct Thin1 {
+  using Value_t = double;
```

Comment:
this is an energy value. Not sure if this has `electronVolts` (as ENDF) or `mega(electronVolts)` (as ACE)